umake/frameworks/ide.py

```diff
@@ -211,10 +211,26 @@ class Idea(BaseJetBrains):
 
     def __init__(self, category):
         super().__init__(name=_("Idea"),
-                         description=_("IntelliJ Idea Community Edition"),
+                         description=_("IntelliJ IDEA Community Edition"),
```

Comment:
You are going to impact translation. Remind me when merging to change that in the .po file and reupload it to launchpad. Thanks!